### DISCUSSION TITLE
Class rray<string,string> does not exist

### DISCUSSION THREAD

**Discussion by siekanski** [api-platform/api-platform#2031] [Q&A] [answered]
Hey @dunglas ,

As you can see in the attached picture I am having trouble generating the page. The problem occurs when I want to generate an endpoint to a class named 'PromoMember' via ApiResource

![image](https://user-images.githubusercontent.com/22029522/134645239-f1e45844-c428-45d1-b0bf-dfbf8ecdf1f2.png)

Here is my code for this situation

![image](https://user-images.githubusercontent.com/22029522/134645690-951dc3e2-9534-4691-9f1b-fb402179dfe5.png)

If I use the CreateATLPromoMemberInput class shown in the picture in the same way in another class that generates endpoints via ApiResource then this problem does not occur. I even tried adding just the GET method in itemOperations in the PromoMember class but it still gives an error. I have a feeling that ApiPlatform is not able to work with the PromoMember class name? The whole mechanism works correctly but the only quite serious problem is the inability to generate a page on the local environment. Is it the PromoMember class name that is somehow not supported and is a proprietary name, or is the problem elsewhere?

**Comment by alanpoulain** [comment]
Hello,
Which version of Symfony are you using?

**Reply by siekanski** [reply]
@alanpoulain
I'm using Symfony 5.0.8

**Reply by alanpoulain** [reply]
It will be solved if you use Symfony ≥ 5.2.

**Reply by siekanski** [reply]
@alanpoulain 
It works ! :) Thanks for the help :)

**Comment by alanpoulain** [comment, marked as answer]
Using Symfony ≥ 5.2 solves this issue.
